rust/src/ffi/util.rs

```diff
@@ -17,6 +18,13 @@ use crate::domains::{VectorDomain, AllDomain, BoundedDomain, InherentNullDomain,
 
 use super::any::{AnyMeasurement, AnyTransformation};
 
+#[cfg(not(feature="untrusted"))]
+use std::marker::PhantomData;
+#[cfg(not(feature="untrusted"))]
+pub struct Sequential<T>(PhantomData<T>);
+#[cfg(not(feature="untrusted"))]
+pub struct Pairwise<T>(PhantomData<T>);
```

Comment:
This puzzled me for a minute, until I figured out why the definitions are needed here. Can we move them up to line 12, so it's more clear that this is just a fallback when we don't import the normal definitions?

Reply:
Same here. I was surprised to find compile errors after I messed up my feature flags.